src/framework/preferences/js/UIEnhancer.js

```diff
@@ -43,13 +43,27 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
 
     fluid.defaults("fluid.uiEnhancer", {
         gradeNames: ["fluid.viewComponent"],
+        defaultLocale: "en",
         invokers: {
             updateModel: {
                 func: "{that}.applier.change",
                 args: ["", "{arguments}.0"]
             }
         },
-        userGrades: "@expand:fluid.prefs.filterEnhancerGrades({that}.options.gradeNames)"
+        userGrades: "@expand:fluid.prefs.filterEnhancerGrades({that}.options.gradeNames)",
+
+        distributeOptions: {
+            "uiEnhancer.messageLoader.defaultLocale": {
+                source: "{that}.options.defaultLocale",
+                target: "{that messageLoader}.options.defaultLocale"
+            },
+            // TODO: This needs to be improved as it is static and should be dynamic. Unfortunately the resource loader
+            //       accepts the locale as an option instead of a model value.
+            "uiEnhancer.messageLoader.locale": {
```

Comment:
Note that the upcoming framework will support dynamic relocalisation - however, I think we'll have a lot of conflicts to work through after this branch goes in. https://github.com/fluid-project/infusion/pull/824/files#diff-2829282b19942f93881f61ac487dad30R827

Reply:
@amb26 are you suggesting that we shouldn't merge this branch?

Reply:
Talked ot @amb26 in the [fluid-work channel today](http://irc-logs.fluidproject.org/%23fluid-work/%23fluid-work.2019-07-23.log). The PR can be merged when ready, it is just a note that there will be merge conflicts with the other work.